content/python/concepts/casting/casting.md

```diff
@@ -36,15 +37,15 @@ This will output:
 
 ## Explicit Type Conversion
 
-In explicit type casting (also known as type conversion), the user converts the data type of variable with the help of predefined functions:
+In explicit type casting, the user converts the data type of a variable using a predefined function. This function is typically the constructor of another data type.
 
-- `str()`: This function takes int or float as an argument and converts it to string data type.
-- `int()`: This function takes string or float as an argument converts it to integer data type.
-- `float()`: This function takes integer or string as an argument and converts it to float data type.
+- `str()`: This function takes an integer or float as an argument and converts it to a string.
+- `int()`: This function takes string or float as an argument converts it to an integer.
+- `float()`: This function takes integer or string as an argument and converts it to a float.
 
 ### Type Casting `int` to `str`
```

Comment:
Given I recently filed [a related issue (#486) about the `str()` function](https://github.com/Codecademy/docs/issues/486), I suggest omitting lines 46-114. Perhaps after this PR is merged, you would be interested in being assigned to write the term entry for the `str()` Python function? @kaifee-haque

Reply:
omitted those lines as suggested! @Dusch4593 @kaifee-haque